pep-0514.txt

```diff
@@ -118,17 +138,22 @@ It is not possible to detect side-by-side installations of both 64-bit and
 current user. Python 3.5 and later always uses different Tags for 64-bit and
 32-bit versions.
 
-Environments registered under other Company names must use distinct Tags to
-support side-by-side installations. Tools consuming these registrations are
-not required to disambiguate tags other than by preferring the user's setting.
+The following section describe user-visible information that may be registered.
+For Python 3.5 and earlier, none of this information is available, but
+alternative defaults are specified for the ``PythonCore`` key.
+
+Environments registered under other Company names have no backward compatibility
+requirements and must use distinct Tags to support side-by-side installations.
+Tools consuming these registrations are not required to disambiguate tags other
+than by preferring the user's setting.
 
 Company
 -------
 
 The Company part of the key is intended to group related environments and to
 ensure that Tags are namespaced appropriately. The key name should be
 alphanumeric without spaces and likely to be unique. For example, a trademarked
-name, a UUID, or a hostname would be appropriate::
+name (preferred), a UUID, or a hostname would be appropriate::
 
```

Comment:
I should point out that I don't want to get too hung up on the specifics of how virtualenv might theoretically implement this option. It would be just as easy for us to add a user config file that allowed the user to manually specify "alias -> Python interpreter path" mappings (much like py.exe has in its ini file). And the fact that we've not done that indicates that the whole issue is low-priority. But essentially, this PEP doesn't add any value for virtualenv unless it's better than a "user maintained alias" approach.

(In practice, I don't think it's likely that this will ever be implemented for virtualenv in its current form - it's not clear that the gain is worth it - but virtualenv remains a simple concrete example of a command line interface).